session.go

```diff
@@ -1044,6 +1051,9 @@ func createSession(store kv.Storage) (*session, error) {
 		parser:      parser.New(),
 		sessionVars: variable.NewSessionVars(),
 	}
+	if cache.PreparedPlanCacheEnabled {
```

Comment:
we should initialize `s.preparedPlanCache` when creating a session, not here .

Reply:
Is CreateSession() right place?

Reply:
yes, in `createSession` and `createSessionWithDomain`

Reply:
Thank you for your comments.